server.js

```diff
@@ -10,7 +10,11 @@ var port = process.env.PORT || 3000;
 
 //require express in our app
 var express = require('express'),
-  bodyParser = require('body-parser');
+    bodyParser = require('body-parser');
+    mongoose = require('mongoose');
+    db = require('./models')
```

Comment:
Since you are not re-writting `var` before each variable, these need to be comma separated at the end instead of semi-colons.

```js
var express = require(),
      bodyParser = require(),
      mongoose = require(),
      db = require()
```